packages/houdini-svelte/src/plugin/codegen/routes/index.ts

```diff
@@ -49,6 +55,20 @@ export default async function svelteKitGenerator(
 				}
 			}
 
+			if (
+				// User uses layout/page queries
+				(uniquePageQueries.length > 0 || uniqueLayoutQueries.length > 0) &&
+				// Include path doesn't include js
+				!config.include.some((path) => path.includes('js')) &&
+				// And the plugin isn't configured to run in static mode
+				!plugin_config(config).static
```

Comment:
So even if they don't use any generated load, they still need to have `.js` in their `include` to get session support. So I think something we want something like this:
```suggestion
				// Include path doesn't include js
				!config.include.some((path) => path.includes('js')) &&
				(
				  // And the plugin isn't configured to run in static mode
				  !plugin_config(config).static ||
				  // User uses layout/page queries
				  (uniquePageQueries.length > 0 || uniqueLayoutQueries.length > 0)
				)
```